src/common/ecmdDefines.H

```diff
@@ -40,6 +40,8 @@
 // If you need to add support to a build target, move it out of here and
 // to the section below
 // Base eCMD
+#ifndef ECMD_BUILD_BASE
```

Comment:
What's the need for the #ifndef/#define here?  The header file `#ifndef ecmdDefines_H/#define ecmdDefines_H` for the whole file should prevent any multiple definition.

Reply:
Good point. I have dropped the extra #ifdef condition and updated the commit message accordingly.